### PR TITLE
Import initial content

### PR DESCRIPTION
**Pull request by berrange**

### CODE REVIEW

**Comment by berrange** [comment]
I don't have any iphone to replicate with, but I've tested with Safari browser (and firefox/chrome/epiphany/konqueror too) and don't see any problem

**Comment by mpolednik** [comment]
Tested w/ safari on macos and works, so yeah - only broken on iOS safari.

**Comment by berrange** [comment]
Added the onclick=void(0) workaround to make hover work on iOS.

File: _layouts/default.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	  <li><a href="/contrib">Contribute</a></li>
+	</ul>
+      </div>
+      <div id="content">
+	<h2>{{ page.title }}</h2>
+      {{ content }}
+      </div>
+    </div>
+    <div id="footer" onclick="void(0)">
+      Participants in the project agree to abide by <a href="/conduct">the code of conduct</a>
```
Comment (stu-gott) [inline _layouts/default.html]: I wasn't aware this had been finalized.

File: fonts/license-sil11.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+---
+title: SIL Open Font License, Version 1.1
+layout: default
+---
+
+Copyright (C) 2015 Red Hat, Inc.,
```
Comment (stu-gott) [inline fonts/license-sil11.md]: Copyright Red Hat? In a license file?

File: run.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+title: Get started with KubeVirt
+layout: default
+---
+
+....insert content showing people how to quickly spin up a kubevirt
```
Comment (stu-gott) [inline run.md]: How compatible is github markup with Jekyll. Can docs/getting-started.md be ported from kubevirt/kubevirt?

File: fonts/license-sil11.md (hunk 2 above)
Comment (berrange) [inline fonts/license-sil11.md]: This is the actual license file text that comes with the fonts, so not something we're free to change.

File: run.md (hunk 3 above)
Comment (berrange) [inline run.md]: That is something to be done as a separate pull request - this is just a placeholder until that is done

File: _layouts/default.html (hunk 1 above)
Comment (berrange) [inline _layouts/default.html]: Do you have any actual objection to having this ? A code of conduct is a must have item for any project with a non-negligible number of contributors IMHO.

Comment (fabiand) [inline _layouts/default.html]: We actually merged a code of conduct here: https://github.com/kubevirt/community/blob/master/code-of-conduct.md

Comment (berrange) [inline _layouts/default.html]: Interesting, I didn't know about that doc. Having looked at it though, it is not entirely satisfactory. In particular it gives no clear contact details of whom to speak to when violation occurs - the statement "email the KubeVirt community team" is too vague - who is in this team & where are their contact details. Also while the text about desired behavioral aspects is fine, it is lacking info about expected consequences of violation or responsibilities of people enforcing it. Rather than trying to update the text ourselves to cover this info, IMHO it is better to just adopt the contributor covenant (http://contributor-covenant.org/) as it is well regarded code of conduct used by 1000's of projects, and constantly being improved meaning we can pull in updates to reflect evolving best practices in this area.

Comment (fabiand) [inline _layouts/default.html]: Good to get that feedback. Sounds reasonable to adopt an existing and well formulated document (like the contributor covenant you reference).
At a first glance the document also looks like something we could adopt.

/cc @rmohr

**Comment by berrange** [comment]
Refreshed pages with updated to code of conduct page listed Fabian as the responsible person to contact.

**Comment by fabiand** [comment]
In general looks good to me as a start.

**Comment by rmohr** [comment]
Content looks good to me.

For me the only thing is that I would want us to adopt an existing framework, like https://phlow.github.io/feeling-responsive/. It covers most of the scenarios, which were mentioned on the mailing thread, out of the box and we don't have to come up with our own css, which we can't really test against many devices/browsers.

I don't want to block this, so if others don't feel the same way, go  ahead.

**Comment by fabiand** [comment]
Opinions?

/cc @stu-gott @admiyo @vladikr @davidvossel

**Comment by berrange** [comment]
I really strongly against adopting a pre-existing theme. IME they end up being a greater burden over the long term.

**Comment by rmohr** [comment]
And in my experience, at the end you end up with pretty much what other themes were about, by copy-pasting in from different solutions, in a way less structured or at best equally well structured way. Further there is a reason why css frameworks like [bootrstrap](http://getbootstrap.com/css/) exist. It is pretty much repetitive how things work, so even if we don't choose a preexisting theme, there is absolutely no need to write your own css.

**Comment by berrange** [comment]
Having maintained websites across multiple different publishing systems, I don't agree with that as it is contrary to everything I've experianced. If you build a theme & css together as one, targeted to a specific site, it will result in a smaller, simpler & easier to understand codebase.

**Comment by fabiand** [comment]
There is no agreement yet on how to maintain this side. But merging it for now to open it up to other PRs to improve it.

This merge does not imply switching over kubevirt.io to this new website.

**Comment by fabiand** [comment]
Merging this PR now even if there are some disagreements on how the websit eis getting maintained.

The reason for merging it is to allow content to be merged.

The switch of kubevirt.io over to this side is not yet taking place.
